content/component-examples/fapolicyd/oval/package_fapolicyd_installed.xml

```diff
@@ -0,0 +1,28 @@
+<def-group>
```

Comment:
This doesn't look entirely like valid OVAL. Can you add the correct namespaces and wrapping elements?